exec/vector/src/main/codegen/templates/NullableValueVectors.java

```diff
@@ -327,13 +327,17 @@ public Mutator getMutator(){
     return v;
   }
 
-  public void copyFrom(int fromIndex, int thisIndex, Nullable${minor.class}Vector from){
+  protected void copyFromUnsafe(int fromIndex, int thisIndex, Nullable${minor.class}Vector from){
```

Comment:
Are changes in this class necessary?

Reply:
I added a copyFromSafe() method elsewhere, which is a "safe copy" that should not throw an exception due to overwriting the end of the buffer.  Because I did that, and realizing that copyFrom() is actually UNSAFE (can throw the exception, which new copyFromSafe() is designed to avoid), I decided for code clarity to change the name of this function.  It has been over a year since I wrote this code, and that is what I recall now.